Makefile

```diff
@@ -1005,6 +1005,11 @@ kind/cluster: local-dev/kind
 		&& echo '  extraMounts:'                                                                      >> $$KINDCONFIG \
 		&& echo '  - containerPath: /var/lib/kubelet/config.json'                                     >> $$KINDCONFIG \
 		&& echo '    hostPath: $(HOME)/.docker/config.json'                                           >> $$KINDCONFIG \
+		&& if [ '$(INSECURE_HOST_MOUNT_API_SRC)' = true ]; then \
+		echo '  - containerPath: /lagoon/services/api/src'                                            >> $$KINDCONFIG \
+		&& echo '    hostPath: ./services/api/src'                                                    >> $$KINDCONFIG \
+		&& echo '    readOnly: true'                                                                  >> $$KINDCONFIG \
```

Comment:
does this make the files read only or just specifying a one-way sync? part of the dev process inside the api is compiling typescript which needs write access to `/services/api/src/dist` and `/node-packages`

Reply:
oh, okay. yes, this currently makes everything inside `services/api/src` read-only from inside the pod. does the process doing the recompilation run inside the pod?

Reply:
I thought this might be useful to make it read-only from inside the pod but of course can leave it as read-write if that makes more sense.